### DISCUSSION TITLE
Dual sensor priority

### DISCUSSION THREAD

**Discussion by Richspiral** [j-kaltes/Juggluco#90] [Ideas]
One of the features I like most about this app is the ability to have two sensors running simultaneously. 

As one sensor ends I extend by 12 hours (another great feature!) and add the new sensor to the app. 

This means I have two sensors running for 12 hours during my cross-over period. 

During this crossover the new sensor is sometimes significantly inaccurate. It's not an issue though as it has the 12 hours to warm up, as the old sensor which is accurate continues to run.

However one issue with this otherwise helpful strategy is that the app often prioritises the new sensor and it's inaccurate readings to display on the notification (which also get fed to xdrip.) 

Sometimes it will see-saw between the two readings. 

This creates confusion. 

I therefore wondered if there is a way to prioritise a sensor and it's readings when we have two sensors running simultaneously? I looked in the sensor settings and couldn't find an option. 

If there isn't such a setting would it be possible to add one? 

Thx 
Rich

**Comment by j-kaltes** [comment]
Do you really have seen a see-saw between readings? Normally x-Drip, in an inscrutable manner, creates a point every 5 minutes out of five 1-minutes inputs it receives.
When creating the patched Libre Broadcast, Juggluco skips all data less than 55 seconds from the previously send broadcast. Mostly these are from a different sensor, because they send their data more than 5 seconds apart from each other.
You can determine from which sensor is sent, by turning off the patched Libre Broadcast and turning it on again just after the sensor, you don't want, send a new value.

**Reply by Richspiral** [reply]
Thanks for the reply and for the tip about hour to prioritise a sensor on xdrip. Will try that next time.
